src/network/clientopcodes.cpp

```diff
@@ -56,72 +56,72 @@ const ToClientCommandHandler toClientCommandTable[TOCLIENT_NUM_MSG_TYPES] =
+	{ "TOCLIENT_SHOW_FORMSPEC",                 TOCLIENT_STATE_CONNECTED, &Client::handleCommand_ShowFormSpec }, // 0x44
+	{ "TOCLIENT_MOVEMENT",                      TOCLIENT_STATE_CONNECTED, &Client::handleCommand_Movement }, // 0x45
+	{ "TOCLIENT_SPAWN_PARTICLE",                TOCLIENT_STATE_CONNECTED, &Client::handleCommand_SpawnParticle }, // 0x46
+	{ "TOCLIENT_ADD_PARTICLESPAWNER",           TOCLIENT_STATE_CONNECTED, &Client::handleCommand_AddParticleSpawner }, // 0x47
+	null_command_handler,
+	{ "TOCLIENT_HUDADD",                        TOCLIENT_STATE_CONNECTED, &Client::handleCommand_HudAdd }, // 0x49
+	{ "TOCLIENT_HUDRM",                         TOCLIENT_STATE_CONNECTED, &Client::handleCommand_HudRemove }, // 0x4a
+	{ "TOCLIENT_HUDCHANGE",                     TOCLIENT_STATE_CONNECTED, &Client::handleCommand_HudChange }, // 0x4b
+	{ "TOCLIENT_HUD_SET_FLAGS",                 TOCLIENT_STATE_CONNECTED, &Client::handleCommand_HudSetFlags }, // 0x4c
+	{ "TOCLIENT_HUD_SET_PARAM",                 TOCLIENT_STATE_CONNECTED, &Client::handleCommand_HudSetParam }, // 0x4d
+	{ "TOCLIENT_BREATH",                        TOCLIENT_STATE_CONNECTED, &Client::handleCommand_Breath }, // 0x4e
+	{ "TOCLIENT_SET_SKY",                       TOCLIENT_STATE_CONNECTED, &Client::handleCommand_HudSetSky }, // 0x4f
+	{ "TOCLIENT_OVERRIDE_DAY_NIGHT_RATIO",      TOCLIENT_STATE_CONNECTED, &Client::handleCommand_OverrideDayNightRatio }, // 0x50
+	{ "TOCLIENT_LOCAL_PLAYER_ANIMATIONS",       TOCLIENT_STATE_CONNECTED, &Client::handleCommand_LocalPlayerAnimations }, // 0x51
+	{ "TOCLIENT_EYE_OFFSET",                    TOCLIENT_STATE_CONNECTED, &Client::handleCommand_EyeOffset }, // 0x52
+	{ "TOCLIENT_DELETE_PARTICLESPAWNER",        TOCLIENT_STATE_CONNECTED, &Client::handleCommand_DeleteParticleSpawner }, // 0x53
+	{ "TOCLIENT_CLOUD_PARAMS",                  TOCLIENT_STATE_CONNECTED, &Client::handleCommand_CloudParams }, // 0x54
+	{ "TOCLIENT_FADE_SOUND",                    TOCLIENT_STATE_CONNECTED, &Client::handleCommand_FadeSound }, // 0x55
+	{ "TOCLIENT_UPDATE_PLAYER_LIST",            TOCLIENT_STATE_CONNECTED, &Client::handleCommand_UpdatePlayerList }, // 0x56
+	{ "TOCLIENT_MODCHANNEL_MSG",                TOCLIENT_STATE_CONNECTED, &Client::handleCommand_ModChannelMsg }, // 0x57
+	{ "TOCLIENT_MODCHANNEL_SIGNAL",             TOCLIENT_STATE_CONNECTED, &Client::handleCommand_ModChannelSignal }, // 0x58
+	{ "TOCLIENT_NODEMETA_CHANGED",              TOCLIENT_STATE_CONNECTED, &Client::handleCommand_NodemetaChanged }, // 0x59
+	null_command_handler,
+	null_command_handler,
+	null_command_handler,
+	null_command_handler,
+	null_command_handler,
+	null_command_handler,
+	{ "TOCLIENT_SRP_BYTES_S_B",                 TOCLIENT_STATE_NOT_CONNECTED, &Client::handleCommand_SrpBytesSandB }, // 0x60
+	{ "TOCLIENT_FORMSPEC_PREPEND",              TOCLIENT_STATE_CONNECTED, &Client::handleCommand_FormspecPrepend }, // 0x61,
```

Comment:
Can't see your changes - too many unrelated whitespace changes.

Reply:
Sorry about that, I indented all the other lines to make up for the extra large length of the network packet in question. :)